tests/unit/components/pacbio/PacbioSampleMetadataEdit.spec.js

```diff
@@ -1,78 +1,85 @@
-import { mount, store } from '@support/testHelper'
-import PacbioSampleMetadataEdit from '@/components/pacbio/PacbioSampleMetadataEdit'
-import PacbioSampleFactory from '@tests/factories/PacbioSampleFactory.js'
+import { mount, createTestingPinia, flushPromises } from '@support/testHelper.js'
+import PacbioSampleMetadataEdit from '@/components/pacbio/PacbioSampleMetadataEdit.vue'
+import { usePacbioRequestsStore } from '@/stores/pacbioRequests.js'
+import PacbioRequestFactory from '@tests/factories/PacbioRequestFactory.js'
 
-const pacbioSampleFactory = PacbioSampleFactory()
+const pacbioRequestFactory = PacbioRequestFactory({ includeRelationships: false })
+
+const mockShowAlert = vi.fn()
+vi.mock('@/composables/useAlert', () => ({
+  default: () => ({
+    showAlert: mockShowAlert,
+  }),
+}))
+
+/**
+ * Helper method for mounting a component with a mock instance of pinia, with the given props.
+ * This method also returns the wrapper and the store object for further testing.
+ *
+ * @param {*} - params to be passed to the createTestingPinia method for creating a mock instance of pinia
+ * which includes
+ * state - initial state of the store
+ * stubActions - boolean to stub actions or not.
```

Comment:
worth adding an extra comment in here to say that if you do not set this to false it will stub all of the actions and could cause errors.